samples/apps/copilot-chat-app/webapi/DeploymentTemplates/DeploySK-Existing-AOAI.ps1

```diff
@@ -1,6 +1,6 @@
 <#
```

Comment:
[very nit] It would be okay to expand the filename here: `DeploySK-Existing-AzureOpenAI.ps1`. (same commend for all instances of "aoai" in filenames.